src/renderer/root/store/configureStore.js

```diff
@@ -14,16 +14,18 @@ export default function configureStore(history) {
 
   const middleware = [thunk, sagaMiddleware, routerMiddleware(history)];
 
-  const composeEnhancers = process.env.NODE_ENV === 'production'
-    ? identity
-    : require('redux-devtools-extension').composeWithDevTools;
+  const composeEnhancers =
+    process.env.NODE_ENV === 'production'
+      ? identity
+      : require('redux-devtools-extension').composeWithDevTools;
 
   const enhancers = composeEnhancers(applyMiddleware(...middleware));
-  const store = createStore(connectRouter(history)(reducers), initialState, enhancers);
+  const store = createStore(reducers(history), initialState, enhancers);
```

Comment:
To match the documentation, perhaps we should now call this `createRootReducer` instead of `reducers` since it's now a function used to create the root reducer.